backend/DB.ts

```diff
@@ -32,10 +33,14 @@ export const dbReady = new Promise<void>((resolve, reject) => {
         });
 
         console.log("Running database migrations");
+        // umzug.down({to: 0}) // undo all migrations
```

Comment:
All the DB.ts code changes were done pairing with Leah to debug the migrations and seeds.  I was having a problem generating and testing a new user and we discovered that we needed to clear the SequelizeMeta table.  I'm happy to figure out the umzug CLI command, but I think it will take some work and maybe should be on a separate ticket?  

The previous comments were done with Leah to help my understanding and I don't think we need them.

Reply:
Okay we can do it in a future PR